components/atom/input/src/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import MoleculeField from '@s-ui/react-molecule-field'
+import FormInput from '@s-ui/react-form-input'
+
+const AtomInput = ({
+  id,
+  label,
+  successText,
+  errorText,
+  helpText,
+  inline,
+  ...props
+}) => {
+  const newProps = {...props}
```

Comment:
this is the same as `const newProps = props`, you could use `props` straight forward

Reply:
I don't think so because `props` is immutable.